docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,34 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Correlations editor in Explore
+
+<!-- Kristina Durivage -->
+<!-- OSS, Enterprise -->
+<!-- already in on-prem -->
+
+<Date>
+
+_Available in public preview in Grafana Cloud_
+
+Creating correlations has just become easier. Try out our new correlations editor in **Explore** by selecting the **+ Add > Add correlation** option from the top bar or from the command palette. The editor shows all possible places where you can place data links and guides you through building and testing target queries. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/correlations/).
+
+To enable this feature, contact Grafana Support.
+
+## Create correlations for provisioned data sources
+
+<!-- Piotr Jamróz -->
+<!-- OSS, Enterprise -->
+<!-- already in on-prem -->
+
+<Date>
```

Comment:
Looking for an approximate release date here

Reply:
It was made available for fast channel on 2023/09/13.